<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTool</class>
 <widget class="QWidget" name="MapTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>46</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MapTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
     QWidget#MapTool{
     border: 1px solid rgb(112,166,246);
     background:rgb(00,00,00);
     }
     QPushButton
     {
     width: 42px;
     height: 42px;
     border:none;
     border-image:url(:/images/btn_normal.png);
     background:transparent;
     color:#ffffff;
     }
     QPushButton:hover
     {
     border:none;
     border-image:url(:/images/btn_hover.png);
     background:transparent;
     }
     QPushButton:pressed{
     border-image:url(:/images/btn_pressed.png);
     background:transparent;
     }
     QPushButton:disabled{
     border-image:url(:/images/btn_disabled.png);
     background:transparent;
     }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="arrowPushButton">
     <property name="toolTip">
      <string>箭头</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../MapMiddleware.qrc">
       <normaloff>:/images/Map_Arrow.png</normaloff>:/images/Map_Arrow.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="roamingPushButton">
     <property name="toolTip">
      <string>漫游</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../MapMiddleware.qrc">
       <normaloff>:/images/Map_Roam.png</normaloff>:/images/Map_Roam.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomInPushButton">
     <property name="toolTip">
      <string>缩小</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../MapMiddleware.qrc">
       <normaloff>:/images/Map_Zoomin.png</normaloff>:/images/Map_Zoomin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zoomOutPushButton">
     <property name="toolTip">
      <string>放大</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../MapMiddleware.qrc">
       <normaloff>:/images/Map_Zoomout.png</normaloff>:/images/Map_Zoomout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="gridPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>网格</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../MapMiddleware.qrc">
       <normaloff>:/images/Map_Grid.png</normaloff>:/images/Map_Grid.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../MapMiddleware.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
